.env.example

```diff
@@ -1,8 +1,8 @@
-DATABASE_URL=
-OTEL_EXPORTER_OTLP_ENDPOINT=
-MNEMONIC_FILE=
-LISTEN_ADDRESS=
-RPC_URL_HTTP=
-RPC_WS_HTTP=
-CONFIG_FILE=
-DB_REFRESH_INTERVAL_SECONDS=
+DATABASE_URL="postgres://paymaster:paymaster@localhost:5432/paymaster"
+# MNEMONIC_FILE = "./tilt/secrets/mnemonic"
+# LISTEN_ADDRESS
+# DB_REFRESH_INTERVAL_SECONDS=
+# OTEL_EXPORTER_OTLP_ENDPOINT=
+# RPC_WS_HTTP=
+# CONFIG_FILE=
+# RPC_URL_HTTP=
```

Comment:
now that Tilt doesn't need this file, it doesn't feel very useful
maybe in the future it'd be more useful to have something like
.env.testnet.example
.env.mainnet.example